cmd/juju/status/status_test.go

```diff
@@ -3239,6 +3520,28 @@ func (as addService) step(c *gc.C, ctx *context) {
 	}
 }
 
+type addServiceBind struct {
```

Comment:
There doesn't need to be a whole new struct. Just add bindings to the existing addService struct.
Since we now call services applications, as a drive by, addService can be renamed to addApplication